### ISSUE TITLE
Do optimizations in frequently called RMIs

### ISSUE THREAD

**Issue by jinbpark** [Samsung/islet#150]
- Some RMI commands (e.g., REC_ENTER) are called lots of times so might be critical to overall performance. Make such jobs as fast as possible.

**Comment by zpzigi754** [comment]
I totally agree with this. I think that making `rmm` with less exits itself could be a good research direction.

**Comment by jinbpark** [comment]
FYI:  This idea came from @bokdeuk-jeong .
